cif_core.dic

```diff
@@ -18456,6 +18480,11 @@ save_publ_author.id
     _type.source                  Related
     _type.container               Single
     _type.contents                Word
+    _method.purpose               Definition
+    _method.expression
+;
+    _enumeration.default = Current_Row(publ_author) + 1
```

Comment:
The `Unique_id` function (https://github.com/COMCIFS/dREL/pull/10) might be more suitable here.

If not, the `Current_row()` call should be updated to reflect the changes implemented in https://github.com/COMCIFS/dREL/pull/9.